src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -227,51 +227,28 @@ Copyright (c) .NET Foundation. All rights reserved.
   <UsingTask TaskName="ResolvePublishAssemblies" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
   <Target Name="RunResolvePublishAssemblies"
           DependsOnTargets="ComputePrivateAssetsPackageReferences;
-                            ComputePackagesToBeFiltered;
                             _DefaultMicrosoftNETPlatformLibrary">
 
     <ResolvePublishAssemblies ProjectPath="$(MSBuildProjectFullPath)"
                               AssetsFilePath="$(ProjectAssetsFile)"
-                              FilterProjectAssetsFile="$(FilterProjectAssetsFile)"
                               TargetFramework="$(TargetFrameworkMoniker)"
                               RuntimeIdentifier="$(RuntimeIdentifier)"
                               PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
                               PrivateAssetsPackageReferences="@(PrivateAssetsPackageReference)"
+                              FilterProjectFiles="$(FilterProjectFiles)"
                               PreserveCacheLayout="$(PreserveCacheLayout)">
 
       <Output TaskParameter="AssembliesToPublish" ItemName="ResolvedAssembliesToPublish" />
-
+      <Output TaskParameter="PackagesPublished" ItemName="ResolvedPackagesPublished" />
     </ResolvePublishAssemblies>
-
+    
+    <ItemGroup>
+      <AllResolvedPackagesPublished Include="@(ResolvedPackagesPublished)"
```

Comment:
Why do we need this extra Item?  Why can't the calling code just use `@(ResolvedAssembliesToPublish)`?